backend/accounts/transaction_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package accounts
+
+import (
+	"testing"
+	"time"
+
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOrderedTransactions(t *testing.T) {
+	tt := func(t time.Time) *time.Time { return &t }
+	fee := coin.NewAmountFromInt64(1)
+	txs := []*TransactionData{
+		{
+			Timestamp: tt(time.Date(2020, 9, 15, 12, 0, 0, 0, time.UTC)),
+			Height:    15,
```

Comment:
Would be nice to sneak in an entry with 0 height somewhere in between.

Reply:
done